databox/api/src/Controller/Admin/AlternateUrlCrudController.php

```diff
@@ -30,27 +31,17 @@ public function configureCrud(Crud $crud): Crud
     public function configureFilters(Filters $filters): Filters
     {
         return $filters
-            ->add(EntityFilter::new('workspace'));
+            ->add(EntityFilter::new('workspace'))
+            ->add(TextFilter::new('type'))
+        ;
     }
 
     public function configureFields(string $pageName): iterable
     {
-        $workspace = AssociationField::new('workspace');
-        $type = TextField::new('type');
-        $label = TextField::new('label');
-        $id = IdField::new();
-        $createdAt = DateTimeField::new('createdAt');
-
-        if (Crud::PAGE_INDEX === $pageName) {
-            return [$id, $workspace, $type, $label];
-        } elseif (Crud::PAGE_DETAIL === $pageName) {
-            return [$id, $type, $label, $createdAt, $workspace];
-        } elseif (Crud::PAGE_NEW === $pageName) {
-            return [$workspace, $type, $label];
-        } elseif (Crud::PAGE_EDIT === $pageName) {
-            return [$workspace, $type, $label];
-        }
-
-        return [];
+        yield IdField::new();
+        yield AssociationField::new('workspace');
+        yield TextField::new('type');
+        yield TextField::new('label');
+        yield DateTimeField::new('createdAt');
```

Comment:
`createdAt` should not be editable (please apply to others)

```suggestion
        yield DateTimeField::new('createdAt')
            ->hideOnForm();
```